docs/common/low_power.md

```diff
@@ -54,43 +51,43 @@ CONFIG_LTE_EDRX_REQ_VALUE_LTE_M="0000"  # eDRX interval 5.12 s for LTE-M
 CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0000"  # eDRX interval 5.12 s for NB-IoT
 ```
 
-eDRX is used to periodically sleep during the PSM Active Timer. With the default settings, the device will monitor for downlink data every 5.12 seconds for the duration of the Active Timer (60 seconds by default) in case the cloud sends data to the device.
+eDRX is used to periodically sleep during the PSM Active Timer. With the default settings, the device monitors for downlink data every 5.12 seconds for the duration of the Active Timer (60 seconds by default) in case the cloud sends data to the device.
```

Comment:
```suggestion
eDRX is used to periodically sleep during the PSM active timer. With the default settings, the device monitors for downlink data every 5.12 seconds for the duration of the Active Timer (60 seconds by default) in case the cloud sends data to the device.
```